src/navigations/HomeBottomTabs.js

```diff
@@ -276,6 +109,7 @@ function HomeBottomTabs({navigation}) {
 
   return (
     <SafeAreaView style={{flex: 1, backgroundColor: COLORS.white}}>
+      <Loading visible={isLoadingFetchingChannelDetail} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A loading spinner has been added which is visible when `isLoadingFetchingChannelDetail` is true. This is a good practice for improving user experience by providing feedback during potentially long operations.
```diff
+ <Loading visible={isLoadingFetchingChannelDetail} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->